src/test/__mocks__/mockFormDefinition.json

```diff
@@ -65,10 +65,45 @@
         {
           "type": "TIME_OF_DAY",
           "required": false,
-          "linkId": "time-of-day-1",
-          "text": "Time of Day",
+          "linkId": "time_1",
+          "text": "Time of Day 1",
           "helperText": "This is helper text"
         },
+        {
+          "type": "TIME_OF_DAY",
+          "required": false,
+          "linkId": "time_2",
+          "text": "Time of Day 2",
+          "helperText": "This is helper text"
+        },
+        {
+          "text": "Duration",
+          "type": "INTEGER",
+          "component": "MINUTES_DURATION",
+          "linkId": "duration_formula",
+          "readOnly": true,
+          "autofillValues": [
+            {
+              "formula": "ABS(time_1 - time_2)",
+              "autofillWhen": [],
+              "autofillBehavior": "ANY",
+              "autofillReadonly": true
+            }
+          ]
+        },
+        {
+          "text": "<b>Duration</b>: ${FORMAT_DURATION(value)}",
+          "type": "DISPLAY",
+          "linkId": "duration_formula_display",
+          "autofillValues": [
+            {
+              "formula": "ABS(time_1 - time_2)",
+              "autofillWhen": [],
+              "autofillBehavior": "ANY",
+              "autofillReadonly": true
+            }
+          ]
+        },
```

Comment:
I tried putting together @ttoomey's recent work on formulas w/ @martha's recent work on minutes duration input, and it magically worked!! well, magically after this bug fix, but still -cool 😄 



<img width="265" alt="Screenshot 2024-02-22 at 5 45 21 PM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/8ec082d7-6cd0-447b-8f18-bb410106985b">

Reply:
Wonderful!! ✨